app/templates/components/paper-select-menu.hbs

```diff
@@ -3,7 +3,7 @@
   disabled=publicAPI.disabled
   actions=publicAPI.actions
   trigger=(component triggerComponent
-    tagName="md-select"
+    disabledProxy=publicAPI.disabled
```

Comment:
We can pass in `disabled=publicAPI.disabled` here. Why the need for `disabledProxy` naming?